cypress/integration/create.js

```diff
@@ -15,6 +15,17 @@ describe('Create Page', () => {
         );
     });
 
+    it('should put the ArrayInput default value', () => {
+        const currentDate = new Date();
+        const currentDateString = currentDate.toISOString().slice(0, 10);
+        cy.get(CreatePage.elements.input('backlinks[0].date')).should(el =>
+            expect(el).to.have.value(currentDateString)
```

Comment:
I'm afraid this is subject to race conditions, where the `Date` generated in the test and the one generated in the default value won't match.

Reply:
No more than in the first create test at L10